src/server/req_delete.c

```diff
@@ -370,7 +370,7 @@ req_deletejob(struct batch_request *preq)
 	int count = 0;
 	int err = PBSE_NONE;
 
-	jid = preq->rq_ind.rq_delete.rq_objname;
+	snprintf(jid, sizeof(jid), "%s", preq->rq_ind.rq_delete.rq_objname);
```

Comment:
Can preq->rq_ind.rq_delete.rq_objname be NULL?

Reply:
It can't be NULL.  We read the name off the wire and if there is a problem, we will catch it then.